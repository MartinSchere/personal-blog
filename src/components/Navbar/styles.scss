.navbar {
  box-shadow: 0px 4px 20px rgba(196, 196, 196, 0.2);
  height: $navbar-height;
  display: flex;
  justify-content: center;
  align-items: center;
  width: 100%;
  position: fixed;
  left: 0;
  top: calc(-1 * #{$navbar-height});
  z-index: 10;
  background-color: $white;
  transition: 300ms ease;
  visibility: hidden;

  &.active {
    top: 0;
    visibility: visible;
  }
}

.navbar-replacement {
  height: $navbar-height;
}
