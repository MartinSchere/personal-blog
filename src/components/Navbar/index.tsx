import React from 'react';
import "./styles.scss";

type NavbarProps = {

}

const Navbar = (props:NavbarProps) => {
return (
<div>
  Navbar
</div>
);
};

export default Navbar;