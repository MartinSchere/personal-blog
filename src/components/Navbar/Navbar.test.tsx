import React from 'react';
import Navbar from './index';
import renderer from 'react-test-renderer';

describe('Navbar', () => {
it('renders without error', () => {
const tree = renderer
.create(<Navbar></Navbar>)
.toJSON();
expect(tree).toMatchSnapshot();
});
});