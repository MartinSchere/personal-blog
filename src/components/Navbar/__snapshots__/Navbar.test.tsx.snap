// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders without error 1`] = `
<div>
  Navbar
</div>
`;
